.travis.yml

```diff
@@ -1,96 +1,60 @@
 language: cpp
+
+dist: trusty
```

Comment:
Sure, but when there will be a new distro, then making trusty explicit here would not automatically move us to the new one. If we remove it, it would use the latest one always (I assume)